pokemon-showdown

```diff
@@ -128,27 +95,27 @@ if (!process.argv[2] || /^[0-9]+$/.test(process.argv[2])) {
 		console.log('  Displays this reference');
 		break;
 	case 'start':
-		{
-			process.argv.splice(2, 1);
-			setupTSNode();
-			require('module')._load('./server', module, true);
-			break;
-		}
+	{
+		process.argv.splice(2, 1);
+		build();
+		require('module')._load('./dist/server/index.js', module, true);
+		break;
+	}
 	case 'generate-team':
 		{
-			sucraseBuild();
-			var Teams = require('./.sim-dist/teams').Teams;
+			ensureBuilt();
+			var Teams = require('./sim/teams.js').Teams;
 			var seed = process.argv[4] ? process.argv[4].split(',').map(Number) : undefined;
 			console.log(Teams.pack(Teams.generate(process.argv[3], {seed})));
 		}
 		break;
 	case 'validate-team':
 		{
-			sucraseBuild();
-			var Teams = require('./.sim-dist/teams').Teams;
-			var TeamValidator = require('./.sim-dist/team-validator').TeamValidator;
+			ensureBuilt();
+			var Teams = require('./sim/teams.js').Teams;
+			var TeamValidator = require('./sim/team-validator.js').TeamValidator;
 			var validator = TeamValidator.get(process.argv[3]);
-			var Streams = require('./.lib-dist/streams');
+			var Streams = require('./lib/streams.js');
```

Comment:
Do you need to specify `.js` here?

Reply:
Nope. This should be pointing to ./dist/lib, though.

Reply:
Fixed, thanks.